### DISCUSSION TITLE
Tracking of over hours

### DISCUSSION THREAD

**Discussion by Blackjacx** [GothenburgBitFactory/timewarrior#571] [Q&A] [answered]
Is it possible to define somewhere that I have a 40 hour week and also to show each day or week if I fulfilled my working hours and eventually if I made overhours?

**Comment by lauft** [comment]
Currently, Timewarrior has no configuration for this. (@kajmaj87 proposed something like this in #532, but there is no PR submitted until now. 😔)

Another solution to this would be to use a custom report. You can find available tools on [the tools page](https://timewarrior.net/tools/). Maybe [this one](https://github.com/guludo/timewarrior-balance) has already everything you need. 🤔 

Otherwise, [this python package](https://pypi.org/project/timew-report/) might be a good starting point for writing your own custom report.

**Reply by kajmaj87** [reply]
Sorry guys, I needed this for my work reports (and fast) and went for a bunch of custom python parsers. I could share those if you're interested but I'm not sure if this would be helpful (they are also quite ugly and written mostly by AI)

**Reply by Blackjacx** [reply]
Nice thank yoiu for ll the links. I wasn't super aware of al the extensions anymore 👍 I think there will be something.

**Reply by Blackjacx** [reply, marked as answer]
I started to use the [balance](https://github.com/guludo/timewarrior-balance) plugin, which does exactly what I need 👍
